telemetry/definitions/commonDefinitions.json

```diff
@@ -2174,6 +2180,154 @@
                 }
             ]
         },
+        {
+            "name": "app_openLocalInvokeWebView",
+            "description": "Click open Local invoke WebView button in Application Builder",
+            "metadata": [
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "source",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "app_openRemoteInvokeWebView",
+            "description": "Click open Remote invoke WebView button in Application Builder",
+            "metadata": [
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "source",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "app_openTemplate",
+            "description": "Click open template button in Application Builder",
+            "metadata": [
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "source",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "app_walkthroughInstallAwsCli",
```

Comment:
all of these `app_walkthroughInstall...` events have a `source` field. So instead of repeating "walkthrough" in the event name, you could set `source=walkthrough` in code. That would allow the events to be re-used for other sources, besides only "walkthrough".

You may also consider generalizing a bit further to:

    app_installTool

which could use the `tool` field to store the actual tool being installed.

https://github.com/aws/aws-toolkit-common/blob/098c0f5911ba20849c141bd55c56eb317bb0e6eb/telemetry/definitions/commonDefinitions.json#L1617